### PR TITLE
Correct Pix2Pix example

### PR DESCRIPTION
**Pull request by patrickvonplaten**
- no advertisement of revision -> it'll be deprecated soon
- by default safety checker should be used

### CODE REVIEW

File: docs/source/en/api/pipelines/stable_diffusion/pix2pix.mdx [hunk 1]
```diff
@@ -45,9 +45,7 @@ import torch
 from diffusers import StableDiffusionInstructPix2PixPipeline
 
 model_id = "timbrooks/instruct-pix2pix"
-pipe = StableDiffusionInstructPix2PixPipeline.from_pretrained(
-    model_id, torch_dtype=torch.float16, revision="fp16", safety_checker=None
-).to("cuda")
+pipe = StableDiffusionInstructPix2PixPipeline.from_pretrained(model_id, torch_dtype=torch.float16).to("cuda")
```
Comment (patrickvonplaten) [inline docs/source/en/api/pipelines/stable_diffusion/pix2pix.mdx:48]:
```suggestion
pipe = StableDiffusionInstructPix2PixPipeline.from_pretrained(model_id, torch_dtype=torch.float16, safety_checker=None).to("cuda")
```

File: docs/source/en/api/pipelines/stable_diffusion/pix2pix.mdx [hunk 2]
```diff
@@ -46,7 +46,7 @@ from diffusers import StableDiffusionInstructPix2PixPipeline
 
 model_id = "timbrooks/instruct-pix2pix"
 pipe = StableDiffusionInstructPix2PixPipeline.from_pretrained(
-    model_id, torch_dtype=torch.float16, revision="fp16", safety_checker=None
+    model_id, torch_dtype=torch.float16, safety_checker=None
```
Comment (patil-suraj) [inline docs/source/en/api/pipelines/stable_diffusion/pix2pix.mdx:49]: `safety_checker` checker is still `None` here.

Comment (Suhail) [reply docs/source/en/api/pipelines/stable_diffusion/pix2pix.mdx:49]: was about to comment the same thing!

Comment (patrickvonplaten) [reply docs/source/en/api/pipelines/stable_diffusion/pix2pix.mdx:49]: Yeah let's open a new PR to fix it

**Pull request merged**